x-pack/plugins/security_solution/common/endpoint/models/policy_config.ts

```diff
@@ -29,6 +29,9 @@ export const policyFactory = (): PolicyConfig => {
         mode: ProtectionModes.prevent,
         supported: true,
       },
+      memory: {
```

Comment:
i think you may also need to add fields for memory in the other two functions: `policyWithoutPaidFeatures` and `policyFactoryWithSupportedFeatures`

Reply:
Thanks!. I'll add them there